src/js/app.js

```diff
@@ -53,13 +65,22 @@ app.controller('walletRecoveryCtrl', function($scope, $modal, $rootScope, $log,
             defaultApiSecret: "MY_APISECRET"
         },
         {
-            name: "Bitpay Insight",
+            name: "Insight Data Service",
             value: "insight_bitcoin_service",
             apiKeyRequired: false,
             apiSecretRequired: false
-        },
+        }
     ];
 
+    if (window.APPCONFIG.RECOVER_LITECOIN) {
+        $scope.recoveryNetwork = {name: "Litecoin", value: "ltc", testnet: false, insightHost: "https://insight.litecore.io/api", recoverySheet: false};
+
+        // remove blocktrail
+        $scope.dataServices.shift()
```

Comment:
Blocktrail won't work for non tbtc/btc, so remove the option when recovering litecoin